src/test/java/com/icthh/xm/ms/entity/web/rest/XmEntityResourceExtendedIntTest.java

```diff
@@ -567,226 +567,6 @@ private XmEntity convertJsonToObject(String json) throws IOException {
         return objectMapper.readValue(json, XmEntity.class);
     }
 
-    @Test
-    public void createXmEntityWithTagsAttachmentsLocations() throws Exception {
```

Comment:
@osiievska23 why this test was removed?

Reply:
This test involves elasticsearch usage and was moved to XmEntityResourceExtendedElasticsearchTest.class

Reply:
done